Sources/SwiftProtobuf/MathUtils.swift

```diff
@@ -12,6 +12,12 @@
 ///
 // -----------------------------------------------------------------------------
 
+#if os(Linux)
+// Linux doesn't seem to define these by default.
```

Comment:
Added a comment linking to my bug report at
  https://bugs.swift.org/browse/SR-4198